jdaviz/configs/imviz/tests/test_parser_roman.py

```diff
@@ -8,18 +8,20 @@
 
 @pytest.mark.parametrize(
     ('ext_list', 'n_dc'),
-    [(None, 5),
+    [(None, 1),
      ('data', 1),
      (['data', 'var_rnoise'], 2)])
 def test_roman_wfi_ext_options(imviz_helper, roman_imagemodel, ext_list, n_dc):
     imviz_helper.load_data(roman_imagemodel, data_label='roman_wfi_image_model', ext=ext_list)
     dc = imviz_helper.app.data_collection
     assert len(dc) == n_dc
 
-    if ext_list is None:
+    if ext_list == '*':
```

Comment:
should this be covered in the test matrix?

Reply:
I'm going to say let's worry about coverage here once we have functionality that requires this behavior.